### ISSUE TITLE
missing <emu-mods>

### ISSUE THREAD

**Issue by jmdyck** [bterlson/ecmarkup#47]
ecmarkdown converts
```
|Initializer_opt|
```
to
```
<emu-nt optional>Initializer</emu-nt>
```
which ecmarkup converts to
```
<emu-nt optional="">Initializer</emu-nt>
```
but this is missing
```
<emu-mods><emu-opt>opt</emu-opt></emu-mods>
```
-------------
Similarly,
```
|Pattern[U]|
```
converts to
```
<emu-nt params="U">Pattern</emu-nt>
```
which is missing
```
<emu-mods><emu-params>[U]</emu-params></emu-mods>
```

**Comment by bterlson** [comment]
I was considering actually changing pipes to be non-terminal references, ie. `|FunctionExpression|` --> `<emu-xref href="#prod-FunctionExpression"></emu-xref>`. Maybe we could then get rid of the mods and leave it up to the emu-production styles to show or hide the them.

**Comment by bterlson** [comment]
Fixed in 2.7.2.
